setup.cfg

```diff
@@ -103,6 +103,12 @@ core =
 # yamllint must remain optional
 yamllint =
   yamllint >= 1.25.0  # GPLv3
+test =
```

Comment:
I don't think these belong in extras. It's absolutely useless for the end-users while being exposed as a public API for installing loosened test deps.